recipes/default.rb

```diff
@@ -64,6 +65,9 @@
   librenms_phpconf = '/etc/php.d/librenms.ini'
   rrdcached_config = '/etc/sysconfig/rrdcached'
 
+  include_recipe 'yum-epel'
+  node.default['yum']['epel']['enabled'] = true
```

Comment:
@nicolasglab you should add this before before including the the recipe (Better in the cookbook attribute)

Reply:
done.